playbooks/roles/l2tp-ipsec/templates/instructions.md.j2

```diff
@@ -94,6 +101,13 @@ L2TP/IPsec connections are not supported out-of-the-box on most Linux distributi
 
 <a name="ios"></a>
 ### iOS ###
+
+There are two ways to configure the iOS L2TP VPN to connect to this Streisand server. The easiest is to install an iOS *profile* containing the VPN definition. You can also set it up manually.
+
+The profile is [available here](streisand.mobileconfig).
```

Comment:
We should include additional instructions here. Maybe something like...

"Tap this link from your iOS device to begin the setup process."